TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/ParkToObzoneWithoutRoadrunner.java

```diff
@@ -0,0 +1,27 @@
+package org.firstinspires.ftc.teamcode.opmodes;
+
+import com.arcrobotics.ftclib.command.CommandScheduler;
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.OpMode;
+
+import org.firstinspires.ftc.teamcode.Robot;
+import org.firstinspires.ftc.teamcode.subsystems.DriveSubsystem;
+
+@Autonomous(name = "Autonomous", group = "Real")
+public class ParkToObzoneWithoutRoadrunner extends OpMode {
+    Robot robot;
+
+    @Override
+    public void init() {
+        robot = new Robot(this);
+        CommandScheduler.getInstance().reset();
+        CommandScheduler.getInstance().cancelAll();
+
+    }
+
+    @Override
+    public void loop() {
+        robot.run();
+        robot.autoModeDrive(1,0,0,3);
```

Comment:
Again, I said all of this in the robot class, but this won't work because the timer is reset with each iteration. It will drive indefinitely.